docs/providers/aws/guide/serverless.yml.md

```diff
@@ -261,6 +261,7 @@ provider:
   stackParameters:
     - ParameterKey: 'Keyname'
       ParameterValue: 'Value'
+  disableRollback: true # To be used for non-production environment
```

Comment:
Why did you write "for non-production environment" here?

I think this param is to control a choice as to whether to attempt to fix a partial deployment, or to roll it all back. That's not necessarily tied to production or not.

"true" is required to work around AWS issues like https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/573 , whether in production or non-production